libs/heuristomancer/resources/ebnf/bed.ebnf

```diff
@@ -0,0 +1,8 @@
+S = header* data+ trailing?
+header = #"[^\r\n]"+ linebreak
+data = #"[^\s]"+ tab number tab number (tab field)* linebreak
```

Comment:
One thing you might have to watch out for is that heuristomancer doesn't scan the entire file, making it possible for a line to be cut off at the beginning before all of the required fields are specified. This could result in some false negatives. A reasonable heuristic would be to require one full data line followed by zero or more partial data lines.